ember-cli-build.js

```diff
@@ -5,6 +5,9 @@ var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function(defaults) {
   var app = new EmberAddon(defaults, {
     // Add options here
+    babel: {
+      includePolyfill: true
```

Comment:
It should not be needed, but right now without it there is a crash in environments without native WeakMap support (like PhantomJS) :( I am trying to figure out what's happening.